codecarbon/core/cpu.py

```diff
@@ -201,11 +201,16 @@ def _fetch_rapl_files(self):
                     name = f"Processor Energy Delta_{i}(kWh)"
                     i += 1
                 rapl_file = os.path.join(self._lin_rapl_dir, file, "energy_uj")
+                rapl_file_max = os.path.join(
```

Comment:
Would you please add a brief comment to describe why this additional rapl file is needed ?